openeo/rest/job.py

```diff
@@ -173,9 +173,13 @@ def logs(
         :return: A list containing the log entries for the batch job.
         """
         url = f"/jobs/{self.job_id}/logs"
-        logs = self.connection.get(url, params={'offset': offset}, expected_status=200).json()["logs"]
+        logs = self.connection.get(
+            url, params={"offset": offset, "log_level": log_level}, expected_status=200
```

Comment:
openEO API only defines log levels as strings, so `log_level` (which at this point can still be an int) in the `params` should be normalized to one of: "error" "warning" "info" "debug"

Reply:
Added a utility function `log_level_name` that converts log_level to a lowercase string, one of the four supported log levels names:
https://github.com/Open-EO/openeo-python-client/pull/408/commits/73d4ad8ac27c59254d165aa234a01a4e7b45197f